README.md

```diff
@@ -36,29 +36,29 @@ Kedro-Viz is an interactive development tool for building data science pipelines
 - 🔬 Focus mode for modular pipeline visualisation
 - 📊 Rich metadata side panel to display parameters, plots, etc.
 - ♻️ Autoreload on code change
-- 🧪 Supports tracking and comparing runs in a Kedro project
+- 🧪 Supports tracking and comparing runs (a.k.a sessions) in a Kedro project
```

Comment:
In @AntonyMilneQB's [comment here](https://github.com/kedro-org/kedro-viz/issues/764#issuecomment-1061842270), he says, "...and don't use "session" in kedro viz at all."

Based on that, should we even include this `(a.k.a sessions)` language? I don't want it to be confusing, and I fear it might be.

Reply:
ah my intention was to clarify the confusion. This is based on the assumption that people using kedro are more familiar with the term kedro sessions and hence when they read 'runs' they might not relate it being the same as sessions. 
but maybe I need to re-word it better. @AntonyMilneQB ?

Reply:
Understand the good intention of clarifying things here but I agree with @tynandebold that is potentially more confusing. The "session only contains one run" clarification is more of a note for developers rather than something to explain to users. So let's remove it from this list completely and just put it in the contributing guide instead - I'll do a commit to add it.

Reply:
Done in [40f6312](https://github.com/kedro-org/kedro-viz/commit/40f6312a75cb110fe58a4d601827ecb69c1ad694).